components/script/dom/bluetoothuuid.rs

```diff
@@ -356,7 +360,7 @@ fn resolve_uuid_name(
                             _ => unreachable!(),
                         };
                         // Step 4.
-                        return Err(Type(format!("Invalid {} name : ' {} {} {}",
+                        return Err(Type(format!("Invalid {} name : '{}'.\n{} {}",
```

Comment:
It is not possible, because the first format string literal argument for `format!` must contain as many `{}` brackets as the number of the macro parameters.